src/System.Private.CoreLib/src/Internal/Runtime/Augments/RuntimeThread.Unix.cs

```diff
@@ -20,21 +23,33 @@ public sealed partial class RuntimeThread
 
         internal WaitSubsystem.ThreadWaitInfo WaitInfo => _waitInfo;
 
+#if MONO
+        private void Unix_PlatformSpecificInitialize()
```

Comment:
Yeah, we need to undo that change. The plan is to use .Unix.cs file only for unix builds of mscorlib. We don't don't have it used yet so perhaps will be easier for you just use Unix before we land that change.

/cc @luhenry